jupyter_server/gateway/gateway_client.py

```diff
@@ -232,7 +232,7 @@ def _headers_default(self):
         default_value=None,
         allow_none=True,
         config=True,
-        help="""The authorization token used in the HTTP headers. The header will be formatted as:
+        help="""The authorization token used in the HTTP headers. The header will be formatted as::
```

Comment:
As this extra colon added on purpose?

Reply:
Yes, this docstring is copied directly into an RST file, and was causing warnings without the extra colon.